src/xia2/Modules/MultiCrystal/ScaleAndMerge.py

```diff
@@ -1173,6 +1178,38 @@ def cluster_analysis(self) -> None:
         self._cos_angle_clusters = self._mca.cos_clusters
         self._cc_clusters = self._mca.cc_clusters
 
+    def export_merged_mtz(
+        self,
+        expts: ExperimentList,
+        refls: flex.reflection_table,
+        file_name: str,
+        d_min: float,
+    ) -> None:
+        merge = Merge()
+        merge.output_filename = file_name
+        merge.set_d_min(d_min)
+        merge.set_wavelength_tolerance(self._params.wavelength_tolerance)
+        merge.set_r_free_params(self._params.r_free_flags)
+        merge.run(expts, refls)
+
+    def export_merged_wave_mtz(
+        self,
+        data_manager: DataManager,
+        wl: float,
+        output_name: str,
+        d_min: float,
+    ) -> str | None:
+        data = data_manager.data_split_by_wl[wl]
+        if data["expt"]:
+            fmt = "%%0%dd" % (math.log10(len(self._data_manager.wavelengths)) + 1)
+            index = sorted(data_manager.wavelengths.keys()).index(wl)
```

Comment:
```suggestion
        wavelength: float,
        output_name: str,
        d_min: float,
    ) -> str | None:
        data = data_manager.data_split_by_wl[wavelength]
        if data["expt"]:
            fmt = "%%0%dd" % (math.log10(len(self._data_manager.wavelengths)) + 1)
            index = sorted(data_manager.wavelengths.keys()).index(wavelength)
```
Unless using `wl` is an _extremely_ common xia2 idiom?